pyro/ops/linalg.py

```diff
@@ -5,43 +5,6 @@
 import torch
 
 
-def rinverse(M, sym=False):
-    """Matrix inversion of rightmost dimensions (batched).
-
-    For 1, 2, and 3 dimensions this uses the formulae.
-    For larger matrices, it uses blockwise inversion to reduce to
-    smaller matrices.
-    """
-    assert M.shape[-1] == M.shape[-2]
-    if M.shape[-1] == 1:
-        return 1./M
-    elif M.shape[-1] == 2:
-        det = M[..., 0, 0]*M[..., 1, 1] - M[..., 1, 0]*M[..., 0, 1]
-        inv = M.new_empty(M.shape)
-        inv[..., 0, 0] = M[..., 1, 1]
-        inv[..., 1, 1] = M[..., 0, 0]
-        inv[..., 0, 1] = -M[..., 0, 1]
-        inv[..., 1, 0] = -M[..., 1, 0]
-        return inv / det.unsqueeze(-1).unsqueeze(-1)
-    elif M.shape[-1] == 3:
-        return inv3d(M, sym=sym)
-    else:
-        # Use blockwise inversion
```

Comment:
I think it's probably safe to use `torch.inverse` here.

Reply:
Oh, sorry about this. I was unaware of this regression, which is why I made the change.

Reply:
No worries, I'm glad we can now use `torch.inverse()` for large matrices. As usual, we find ourselves in a neglected region of performance space, operating on large batches of small matrices :-/